applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/prj_fast_pair.conf

```diff
@@ -0,0 +1,145 @@
+# The build type configuration supports both configuration channel and MCUmgr DFU transports.
+#
+# Application Configuration
+
+CONFIG_DESKTOP_INIT_LOG_MOTION_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+CONFIG_CAF_INIT_LOG_KEEP_ALIVE_EVENTS=n
+
+CONFIG_DESKTOP_ROLE_HID_PERIPHERAL=y
+CONFIG_DESKTOP_PERIPHERAL_TYPE_MOUSE=y
+CONFIG_DESKTOP_DEVICE_PID=0x52DE
+CONFIG_DESKTOP_DEVICE_GENERATION="FastPair"
+CONFIG_DESKTOP_HID_BOOT_INTERFACE_MOUSE=y
+
+CONFIG_DESKTOP_MOTION_BUTTONS_ENABLE=y
+CONFIG_DESKTOP_MOTION_BUTTONS_UP_KEY_ID=0
+CONFIG_DESKTOP_MOTION_BUTTONS_DOWN_KEY_ID=3
+CONFIG_DESKTOP_MOTION_BUTTONS_LEFT_KEY_ID=2
+CONFIG_DESKTOP_MOTION_BUTTONS_RIGHT_KEY_ID=1
+
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE=n
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_LEDS=y
+
+CONFIG_DESKTOP_DFU_MCUMGR_ENABLE=y
+CONFIG_CAF_INIT_LOG_BLE_SMP_TRANSFER_EVENTS=n
+
+# Align advertising intervals with Fast Pair specification.
+CONFIG_CAF_BLE_ADV_FAST_INT_MIN=0x0040
+CONFIG_CAF_BLE_ADV_FAST_INT_MAX=0x0060
+CONFIG_CAF_BLE_ADV_SLOW_INT_MIN=0x0060
+CONFIG_CAF_BLE_ADV_SLOW_INT_MAX=0x0080
+
+# Disable filter accept list to allow for multiple bonds for a given local identity.
+CONFIG_CAF_BLE_ADV_FILTER_ACCEPT_LIST=n
+CONFIG_CAF_BLE_STATE_MAX_LOCAL_ID_BONDS=3
+
+# Align the advertised TX power with Fast Pair expectations.
+CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL=-17
+
+CONFIG_DESKTOP_BLE_USE_DEFAULT_ID=y
+
+CONFIG_DESKTOP_BLE_PEER_CONTROL=y
+CONFIG_DESKTOP_BLE_PEER_CONTROL_BUTTON=0x0000
+CONFIG_DESKTOP_BLE_PEER_ERASE_ON_START=y
+
+CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S=30
+CONFIG_DESKTOP_BLE_LOW_LATENCY_LOCK=y
+
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT=y
+CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE=y
+
+################################################################################
+# Zephyr Configuration
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1536
+CONFIG_ISR_STACK_SIZE=1536
+CONFIG_MAIN_STACK_SIZE=840
+CONFIG_BT_RX_STACK_SIZE=2048
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=1536
+
+CONFIG_BOOT_BANNER=n
+CONFIG_NCS_BOOT_BANNER=n
+
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_SYSTEM_CLOCK_NO_WAIT=y
+
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+
+CONFIG_GPIO=y
+
+CONFIG_REBOOT=y
+
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+CONFIG_PWM=y
+
+CONFIG_LED=y
+CONFIG_LED_PWM=y
+
+# Enable MCUmgr Bluetooth transport and increase Bluetooth buffers to speed up DFU image transfer.
+CONFIG_MCUMGR_TRANSPORT_BT=y
+CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN=n
+CONFIG_MCUMGR_GRP_IMG_VERSION_CMP_USE_BUILD_NUMBER=y
+
+CONFIG_IMG_MANAGER=y
+CONFIG_MCUBOOT_IMG_MANAGER=y
+
+CONFIG_MCUMGR_GRP_IMG_REJECT_DIRECT_XIP_MISMATCHED_SLOT=y
+
+CONFIG_BT_L2CAP_TX_MTU=260
+CONFIG_BT_BUF_ACL_RX_SIZE=264
```

Comment:
Is there a reason why we increase only to those values instead of increasing it like in the nRF54H configs?
```
# Allow for large Bluetooth data packets.
CONFIG_BT_L2CAP_TX_MTU=498
CONFIG_BT_BUF_ACL_RX_SIZE=502
CONFIG_BT_BUF_ACL_TX_SIZE=251
```

Reply:
Only nRF54H uses the SMP reassembly and configures bigger buffers in nRF Desktop. IIRC, this was done because the nRF54H updates both cores and it could generally transfer bigger update image.